tests/integration/components/curriculum-inventory-sequence-block-header-test.js

```diff
@@ -1,140 +1,97 @@
-import RSVP from 'rsvp';
-import EmberObject from '@ember/object';
 import { setupRenderingTest } from 'ember-qunit';
-import {
-  render,
-  settled,
-  click,
-  findAll,
-  fillIn,
-  triggerEvent
-} from '@ember/test-helpers';
+import { render } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import hbs from 'htmlbars-inline-precompile';
+import { component } from 'ilios/tests/pages/components/curriculum-inventory-sequence-block-header';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 
-const { resolve } = RSVP;
-
-module('Integration | Component | curriculum inventory sequence block header', function(hooks) {
+module('Integration | Component | Curriculum Inventory Sequence Block Header', function(hooks) {
   setupRenderingTest(hooks);
+  setupMirage(hooks);
 
   test('it renders', async function(assert) {
-    const block = EmberObject.create({
-      title: 'Block title'
-    });
+    const block = this.server.create('curriculum-inventory-sequence-block', { title: 'Block title' });
     this.set('sequenceBlock', block);
-    await render(hbs`<CurriculumInventorySequenceBlockHeader
-      @sequenceBlock={{sequenceBlock}}
-      @canUpdate={{true}}
-    />`);
-    assert.dom('.title').hasText(block.title, 'Block title is visible');
-    assert.dom('.editable').exists({ count: 1 }, 'Block title is editable.');
+    await render(
+      hbs`<CurriculumInventorySequenceBlockHeader @sequenceBlock={{this.sequenceBlock}} @canUpdate={{true}} />`
+    );
+    assert.ok(component.title.isVisible);
+    assert.equal(component.title.value, block.title);
+    assert.ok(component.title.isEditable);
   });
 
   test('read-only mode for block in when it can not be updated', async function(assert) {
-    const block = EmberObject.create({
-      title: 'Block title',
-      report: {
-      }
-    });
+    const block = this.server.create('curriculum-inventory-sequence-block', { title: 'Block title' });
     this.set('sequenceBlock', block);
```

Comment:
This works for now, but `block` here isn't actually an EmberData model, it's a mirage value object. Should be `const blockModel = await this.owner.lookup('service:store').find('curriculum-inventory-sequence-block', block.id);` to be test everything as it is for a user.

Reply:
handled. :+1: